cmd/fyne_demo/FyneApp.toml

```diff
@@ -3,10 +3,16 @@
   Name = "Fyne Demo"
   ID = "io.fyne.demo"
   Version = "2.3.0"
-  Build = 10
+  Build = 11
 
 [Development]
   HelperText = "This binary was build with debug symbol"
 
 [Release]
   HelperText = "This binary was build without debug symbol"
+
+[LinuxAndBSD]
```

Comment:
Why not `Freedesktop` here as that the specification they apply to.

Reply:
Andrew and I discussed it on a call and decided that we could more settings there in the future, unrelated to desktop files, it's the current naming. All of our users/developers may or may not know what Freedesktop is.